step-release-vis/src/app/components/environment/environment.ts

```diff
@@ -175,20 +175,21 @@ export class EnvironmentComponent implements OnInit, OnChanges {
     return polygon.highlight ? '1.0' : '0.7';
   }
 
-  polygonMouseEnter(polygon: Polygon, event: MouseEvent): void {
+  enteredPolygon(polygon: Polygon, event: MouseEvent): void {
     this.candidateService.polygonHovered(polygon);
     this.moveTooltip(event);
   }
 
-  polygonMouseLeave(polygon: Polygon): void {
+  leftPolygon(polygon: Polygon): void {
     this.candidateService.polygonUnhovered(polygon);
     this.hideTooltip();
```

Comment:
Remove `this.hideTooltip();` since env mouse leave takes care of that.

Reply:
done